src/DataManagement/BundleData.js

```diff
@@ -89,6 +83,23 @@ class BundleData {
     fs.writeFileSync(path.join(this.baseAreasPath, name, 'items.yml'), '');
     fs.writeFileSync(path.join(this.baseAreasPath, name, 'rooms.yml'), '');
     fs.writeFileSync(path.join(this.baseAreasPath, name, 'npcs.yml'), '');
```

Comment:
Then, since it looks like the goal of this is just to set up a "skeleton" area with a manifest, you could have a method called something like `createBlankArea` that takes the name, title, and suggested level and does the work of lines 77-85 in one go.